core/src/main/java/org/apache/brooklyn/enricher/stock/Enrichers.java

```diff
@@ -293,42 +307,33 @@ protected String getDefaultUniqueTag() {
             return "aggregator:"+publishing.getName();
         }
         public EnricherSpec<?> build() {
-            Predicate<Object> valueFilter;
```

Comment:
Aha the removed lines are redundant because of `Aggregator.getDefaultValueFilter()`.

Possibly worth promoting that method to `AbstractAggregator` but I agree with your analysis.